### DISCUSSION TITLE
Is it possible to apply weighting to focei or saem estimates?

### DISCUSSION THREAD

**Discussion by billdenney** [nlmixr2/nlmixr2#156] [Q&A]
I'm wanting to model the standard deviation of a measurement for a model-based meta-analysis using nlmixr2.  Is it possible to add a weighting factor so that I can weight my measurements by the number of patients for each observation?

**Comment by billdenney** [comment]
As I was thinking about it a bit more, is the answer as simple as using something like the following for my residual error model:

```
    addSdEst <- addSd/sqrt(n)
    est ~ add(addSdEst)
```

**Reply by mattfidler** [reply]
This would only work for `focei` and it isn't clear if it would do so great.

**Reply by mattfidler** [reply]
Although this may change in the future.

**Reply by mattfidler** [reply]
One of the possibilities is to use bayseian priors, which can be specified by adjusting the liklihood but it isn't implemented yet.

**Reply by mattfidler** [reply]
It may be easier to try another approach


```r
   estW ~ est*sqrt(n)
    estW ~ add(addSdEst)
```

Since `n` is already known, multiply `DV` by `sqrt(n)`

Then you can use saem or focei, it doesn't much matter.  The residuals and predictions would have to be adjusted.

**Reply by billdenney** [reply]
FYI, both worked effectively the same way, so I stuck with the version that did not require changing the dataset or predictions.  What are the caveats that make it not "clear if it would do so great"?

**Reply by mattfidler** [reply]
The method has not been tested as extensively.  In some eta/epsilon models it didn't work as well as I would like, there is some things that are likely to be adjusted in the future
